tests/testthat/test-get_publication.R

```diff
@@ -17,3 +17,19 @@ test_that("Retrieve data set list for publication", {
     readRDS("testdata/example_publication_datasets.rds")
   )
 })
+
+test_that("Search works as expected", {
+  expect_equal(
+    get_publications(search = "API") |>
+      dplyr::filter(!grepl("API", title)) |>
+      nrow(),
+    0
+  )
+})
```

Comment:
This test would also pass if the API call returned no rows (without testing the search at all), are we okay with that?

Reply:
Have done a couple of different re-writes.